tools/dump-anonymize.mjs

```diff
@@ -1,10 +1,10 @@
 // usage:
-// $ node dump-anonymize.js <data-dump-folder> <anonymous-output-folder>
+// $ node dump-anonymize.mjs <data-dump-folder> <anonymous-output-folder>
 
-const fs = require('fs');
-const async = require('async');
-const { v4: uuidv4 } = require('uuid');
-const path = require('path');
+import fs from 'node:fs';
+import async from 'async';
+import { v4 as uuidv4 } from 'uuid';
+import path from 'node:path';
```

Comment:
```suggestion
import * as fs from 'node:fs';
import * as async from 'async';
import { v4 as uuidv4 } from 'uuid';
import * as path from 'node:path';
```

Same in other changes.

Reply:
Why? We want to be moving the other direction (away from `import * as ...`) as we move to native ESM.

Reply:
Ok, I was unaware this was possible in ESM. Carry on...

Reply:
Node does some interesting stuff to make this sorta Just Work (https://nodejs.org/api/esm.html#commonjs-namespaces):

> For better compatibility with existing usage in the JS ecosystem, Node.js in addition attempts to determine the CommonJS named exports of every imported CommonJS module to provide them as separate ES module exports using a static analysis process.

See also https://github.com/nodejs/cjs-module-lexer.